docs/content/preview/api/ysql/_index.md

```diff
@@ -14,55 +14,48 @@ aliases:
+## The SQL language
 
 The main components of YSQL include the data definition language (DDL), the data manipulation language (DML), the data control language (DCL), built-in SQL functions, and the PL/pgSQL procedural language for stored procedures. These components depend on underlying features like the data type system (common for both SQL and PL/pgSQL), expressions, database objects with qualified names, and comments. Other components support purposes such as system control, transaction control and performance tuning.
 
-{{< note title="Note" >}}
+| Section | Description |
+| :--- | :--- |
+| [YSQL statements](statements/) | Each statement has its own dedicated page. Each page starts with a formal specification of the syntax: both as a _railroad diagram_; and as a _grammar_ using the PostgreSQL convention. Then it explains the semantics and illustrates the explanation with code examples. |
+| [WITH clause](with-clause/) | Using the WITH clause and common table expressions. |
+| [Built-in SQL functions](exprs/) | YSQL supports all PostgreSQL-compatible built-in functions and operators. |
+| [User-defined subprograms](user-defined-subprograms-and-anon-blocks/) | Functions, procedures, and anonymous blocks. |
+| [Data types](datatypes/) | Data types in YSQL, which supports most PostgreSQL-compatible data types. |
+| [Keywords](keywords) | Lists the keywords in YSQL. |
+| [Reserved names](reserved_names) | Lists names and qualifiers reserved for the system. |
+
+### PostgreSQL documentation
 
 If you don't find what you're looking for in the YSQL documentation, you might find answers in the relevant [PostgreSQL documentation](https://www.postgresql.org/docs/11/index.html). Successive YugabyteDB releases honor PostgreSQL syntax and semantics, although some features (for example those that are specific to the PostgreSQL monolithic SQL database architecture) might not be supported for distributed SQL. The YSQL documentation specifies the supported syntax and extensions.
 
-To find the version of the PostgreSQL processing layer used in YugabyteDB, you can use the `version()` function. The following YSQL query displays only the first part of the returned value:
+To find the version of the PostgreSQL processing layer used in YugabyteDB, use the `version()` function. The following YSQL query displays only the first part of the returned value:
 
 ```plpgsql
 select rpad(version(), 18)||'...' as v;
 ```
 
-For the “latest” release series of YugabyteDB, as reflected in this main documentation URL,
-the query result shows that the PostgreSQL version is 11.2:
+For the latest release series of YugabyteDB, as reflected in this main documentation URL, the query result shows that the PostgreSQL version is 11.2:
```

Comment:
@polarweasel, @ddhodge:

***Please do not merge this change  without having a three-way Alex-Dwight-Bryn discussion.**

I made the change that Dwight now wants to reverse in [[PR #6106](https://github.com/yugabyte/yugabyte-db/pull/6106)](https://github.com/yugabyte/yugabyte-db/pull/6106)—merged 21-Oct-2020. I did it for very good reason. I discussed the proposed plan with Steve Bang (no longer with Yugabyte Inc) and @m-iancu at the time. And they approved it.

There are two separable questions:

- Where should the top of the YSQL API Reference tree be in the overall YugabyteDB hierarchy?

- What should be the hierarchal breakdown under the the YSQL API Reference top-of-tree?

The first question is a non-issue because Googling for « YugabyteDB YSQL documentation » finds the proper page, [[HERE](https://docs.yugabyte.com/preview/api/ysql/)](https://docs.yugabyte.com/preview/api/ysql/), as the top hit. Anybody who wants to know about SQL syntax and semantics (e.g. "How do I achieve _X_?) is well-used to Googling and knows how to compose the search string that best helps them—albeit after some trial-and-error. And once the YSQL API Reference top-of-tree is found, and bookmarked, that's water under the bridge.

The critical concern is the taxonomy of YSQL API topics. The major top-level split is between subtopics like these:

- The SQL Language
- Data types
- Built-in functions and operators
- User-defined subprograms and anonymous blocks—"language SQL" and "language plpgsql"

and so on. Then each has its own breakdown in the normal way. SQL breaks down into subtopics like these:

- SQL statements (syntax and semantics)
- Join methods
- Expression syntax
- WITH clause

and so on. We've just skimmed the surface so far.

It's madness to promote all present and future children of the SQL Language to top-level. Next thing, You'll want to do it, say, for Data types and for User-defined subprograms and anonymous blocks...